fastlane/Fastfile

```diff
@@ -265,6 +269,17 @@ platform :android do
                 end
 
             end
+
+            validateReleaseNotes(releaseNotes: releaseNotes)
+            releaseNotes
+        end
+
+        desc "Validates the release notes to ensure they are suitable for the Play Store"
+        private_lane :validateReleaseNotes do |options|
+            releaseNotesLength = options[:releaseNotes].length
```

Comment:
using `"".length` might not give us the exact match the Play Store uses (500 unicode characters) but given the release notes will be English, that should be close enough in most cases